src/legacy/core_plugins/kibana/common/field_formats/types/date.ts

```diff
@@ -31,42 +41,38 @@ export function createDateFormat(FieldFormat) {
     getParamDefaults() {
       return {
         pattern: this.getConfig('dateFormat'),
-        timezone: this.getConfig('dateFormat:tz')
+        timezone: this.getConfig('dateFormat:tz'),
       };
     }
 
-    _convert(val) {
+    _convert(val: any) {
```

Comment:
Looks lilke you forgot to refactor fn -> object here

Reply:
Done